readme.md

```diff
@@ -124,6 +125,7 @@ Please be aware that videos can be outdated. Always verify them with the docs.
 
 - [R3D4](https://r3d4.fr) - Faucet, ESDT tools, Stats
 - [Convertes for Elrond Blockchain](http://207.244.241.38/elrond-converters) - This page offers an easy to use pack of convertors necessary for interacting with Elrond Blockchain.
+- [Middleman.nft](https://www.middleman-nft.com/) - A secure way to exchange NFTs with a smart contract that acts as a trusted third party. Open source code of the frontend [here](https://github.com/yum0e/front-nft-middleman).
```

Comment:
Hey, thanks for the contribution, the smart contract is nice and in the proper section, but could you create a separate section in the 'community' called 'dApps source code' and link there yours: https://github.com/yum0e/front-nft-middleman? I think the marketplace isn't suitable here, but I think a separate section would be excellent because you have it as an open-source app. The section with open-source dapps will be similar to smart contracts. The `Dev dApps` section is more for the utility dapps, which developers can use in daily work. Thanks.